<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAnaAttrDialog</class>
 <widget class="QDialog" name="CAnaAttrDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="m_pFreeRadioButton">
       <property name="text">
        <string>自由组合</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="m_pADRelyRadioButton">
       <property name="text">
        <string>AD保护采样值</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="m_pSVRlyRadioButton">
       <property name="text">
        <string>SV保护采样值</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="m_pCalRelyRadioButton">
       <property name="text">
        <string>衍生通道</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="m_pDevInfoRadioButton">
       <property name="text">
        <string>设备状态监测通道</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="m_DB0CheckBox">
         <property name="text">
          <string>DB0:可见标志</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB1CheckBox">
         <property name="text">
          <string>DB1:信号上送标志</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB2CheckBox">
         <property name="text">
          <string>DB2:录波标志</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB3CheckBox">
         <property name="text">
          <string>DB3:保护量采样值</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB4CheckBox">
         <property name="text">
          <string>DB4:保护量计算采样值</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB5CheckBox">
         <property name="text">
          <string>DB5:保护量计算向量</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB6CheckBox">
         <property name="text">
          <string>DB6:遥测采样值</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB7CheckBox">
         <property name="text">
          <string>DB7:遥测计算向量</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB8CheckBox">
         <property name="text">
          <string>DB8:遥测功率</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB9CheckBox">
         <property name="text">
          <string>DB9:遥测频率</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB10CheckBox">
         <property name="text">
          <string>DB10:遥测直流</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB11CheckBox">
         <property name="text">
          <string>DB11:遥测档位</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB12CheckBox">
         <property name="text">
          <string>DB12:遥测设备态</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB13CheckBox">
         <property name="text">
          <string>DB13:交流量幅值调节系数</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB14CheckBox">
         <property name="text">
          <string>DB14:交流量谐波计算</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB15CheckBox">
         <property name="text">
          <string>DB15:遥测方向计算</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="m_DB16CheckBox">
         <property name="text">
          <string>DB16:AGC通道(直流)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB17CheckBox">
         <property name="text">
          <string>DB17:运行界面可见标志</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB18CheckBox">
         <property name="text">
          <string>DB18:设备状态监测</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB19CheckBox">
         <property name="text">
          <string>DB19:交流量相位调节系数</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB20CheckBox">
         <property name="text">
          <string>DB20:设备状态监视调节系数</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB21CheckBox">
         <property name="text">
          <string>DB21:备用</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB22CheckBox">
         <property name="text">
          <string>DB22:备用</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB23CheckBox">
         <property name="text">
          <string>DB23:备用</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB24CheckBox">
         <property name="text">
          <string>DB24:备用</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB25CheckBox">
         <property name="text">
          <string>DB25:备用</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB26CheckBox">
         <property name="text">
          <string>DB26:备用</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB27CheckBox">
         <property name="text">
          <string>DB27:备用</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB28CheckBox">
         <property name="text">
          <string>DB28:备用</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB29CheckBox">
         <property name="text">
          <string>DB29:GS接收浮点值(0)/整型值(1)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB30CheckBox">
         <property name="text">
          <string>DB30:GS接收通道</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB31CheckBox">
         <property name="text">
          <string>DB31:SV接收通道</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CAnaAttrDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CAnaAttrDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
